src/utils/getFullName.ts

```diff
@@ -1,23 +1,27 @@
-import isEmpty from "lodash/isEmpty";
 import { nbsp } from "../constants";
 
 type User = {
-    firstName?: string,
-    lastName?: string,
+    firstname?: string;
+    lastname?: string;
+    firstName?: string;
+    lastName?: string;
```

Comment:
Why are there two versions of the same thing, `firstName` and `firstname`? We should pick one and follow it.

Reply:
Unfortunately, we can't choose just one option, because the `Contact` type uses lowercase (`firstname`, `lastname`) while the `Owner` type uses camelcase (`firstName`, `lastName`)

Reply:
Ah okay, in that, maybe split it into two types, Contact and Owner, then say the input type to the function is `Contact|Owner`.

Reply:
fixed